snippets/query.json

```diff
@@ -69,5 +69,12 @@
             "}"
         ],
         "description": "Snippet: Query of type API"
+    },
+    "Query DataItemLink": {
+        "prefix": "tquerydataitemlink",
+        "body": [
+            "\t\t\tDataItemLink = ${1:id} = ${2:parent}.${3:parentid};"
```

Comment:
I'd go with `column` instead of `id` so it's:

```
"\t\t\tDataItemLink = ${1:column} = ${2:parent}.${3:parentColumn};"
```

because `id` is a pretty loaded term in AL.